Vagrantfile

```diff
@@ -95,6 +95,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       app.vm.synced_folder "src/rf/apps", "/opt/app/apps"
     else
       app.vm.synced_folder "src/rf", "/opt/app/"
+      app.vm.synced_folder "#{ENV['HOME']}/.aws", "/var/lib/rf/.aws/",
+        owner: "rf",
```

Comment:
It was a permission issue but I don't remember what. I can try to reproduce it if you want. But the boto config solutions seems like it might be a good bet.